src/Components/Aspire.Microsoft.Azure.Cosmos/AspireMicrosoftAzureCosmosDBExtensions.cs

```diff
@@ -83,6 +83,8 @@ public static class AspireMicrosoftAzureCosmosDBExtensions
         configureSettings?.Invoke(settings);
 
         var clientOptions = new CosmosClientOptions();
+        clientOptions.ApplicationName = CosmosConstants.CosmosApplicationName;
```

Comment:
Do we want to set the same for the the EntityFramework component: https://github.com/dotnet/aspire/tree/main/src/Components/Aspire.Microsoft.EntityFrameworkCore.Cosmos?

Reply:
I am not sure if we really need to set this. When I look at the definition of `CosmosDbContextOptionsBuilder`, I do not see an option to set the application name in the entity framework library. Cc: @kirankumarkolli .

Reply:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.infrastructure.cosmosdbcontextoptionsbuilder?view=efcore-8.0

Am also not seeing direct API for it. 

@ajcvickers thoughts?